### DISCUSSION TITLE
How Score and leaderboard works?

### DISCUSSION THREAD

**Discussion by luigiMinardi** [aradzie/keybr.com#74] [Q&A]
I have 550 lessons done and I'm finally starting to get some good speeds (for my objectives at least), but my score never get to 2k+.

Also I dunno if the scores of past tests adds up to your account score (that I can't see anywere so I can't really know if only the score of the last test matters or if your "account score" is literally what is shown in the typing practice page after you complete a test).

**Comment by luigiMinardi** [comment]
Ok I think I get it, it seems that the key to raise your points is consistency right? 

Edit: But like, I was with 4k rn, then I lost 0.6% accuracity on my last test and now I'm with 779 points and the UI shows that I lost 493 points so I really don't get it.

**Comment by nottud** [comment]
I think the score up and down is comparing with your average score - not your last score.
